worlds/witness/data/static_locations.py

```diff
@@ -460,7 +450,7 @@ def get_id(entity_hex: str) -> int:
     Calculates the location ID for any given location
     """
 
-    return cast(int, static_witness_logic.ENTITIES_BY_HEX[entity_hex]["id"])
+    return cast("int", static_witness_logic.ENTITIES_BY_HEX[entity_hex]["id"])
```

Comment:
What?

Reply:
Not sure. Let me check what the rule is

Reply:
> Apparently it's [TC006](https://docs.astral.sh/ruff/rules/runtime-cast-value/). Not a huge fan to be honest.

Me neither. But oh well, I don't think I will disable it specifically just because I think it's a bit ugly